doc/release/upcoming_changes/23136.performance.rst

```diff
@@ -0,0 +1,17 @@
+``ufunc.at`` can be much faster
+-------------------------------
+Generic ``ufunc.at`` can be up to 9x faster. The conditions for this speedup:
+
+- contiguous arguments
```

Comment:
Is this actually needed?  This release note covers both speedups (no-casting fast path and "super-fast" path?)

Reply:
Correct, it should be "operands are aligned"